code/datums/components/stationloving.dm

```diff
@@ -56,13 +56,16 @@
 
 /datum/component/stationloving/proc/in_bounds()
 	var/static/list/allowed_shuttles = typecacheof(list(/area/shuttle/syndicate, /area/shuttle/escape, /area/shuttle/pod_1, /area/shuttle/pod_2, /area/shuttle/pod_3, /area/shuttle/pod_4))
+	var/static/list/disallowed_centcom_areas = typecacheof(list(/area/abductor_ship, /area/fabric_of_reality, /area/awaymission/errorroom))
 	var/turf/T = get_turf(parent)
 	if (!T)
 		return FALSE
 	var/area/A = T.loc
-	if(istype(A, /area/fabric_of_reality)) // Obviously terrible, just for test merging
```

Comment:
for_testing_do_not_publish_to_live_env_i_mean_it_looking_at_you_tim

Reply:
this has been in the code since May 25 of 2018, so a year + about a week
incredible